compiler/AST/AstToText.cpp

```diff
@@ -1426,50 +1451,56 @@ void AstToText::appendExpr(IfExpr* expr, bool printingType)
 
 void AstToText::appendExpr(LoopExpr* expr, bool printingType)
 {
-  if (expr->forall)
+  std::string start,end;
+  if(expr->forall)
   {
-    if (expr->maybeArrayType)
+    if(expr->maybeArrayType)
```

Comment:
Minor nit, but it would be good to restore the missing space to the above two if statements